Composer/packages/server/src/controllers/asset.ts

```diff
@@ -118,7 +118,23 @@ export async function getTemplateReadMe(req: any, res: any) {
       const moduleURL = `https://registry.npmjs.org/${moduleName}`;
       const response = await fetch(moduleURL);
       const data = await response.json();
-      res.status(200).json(data?.readme || '');
+      // check for readme at root of response obj
+      let readMe = data?.readme;
+
+      // if no readme at root then pull readme from latest published version that has one
+      if (!readMe) {
+        const versionsDict = data?.versions;
+        if (versionsDict && Object.keys(versionsDict).length > 0) {
+          for (const versionKey of Object.keys(versionsDict).reverse()) {
```

Comment:
this is written that the last key in the object returned from Object.keys is the latest. Object.keys() does not guarantee that. What other information is in this payload that can more deterministically find the latest version? Are dates associated with the version anywhere?

Reply:
There is a version string array that is ordered based on last published and that string value actually is the key to the versions dicitonary object so that should be a good way to get latest. Ill make that change now.